packages/sources/cryptocompare-test/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@chainlink/cryptocompare-test-adapter",
+  "version": "0.0.0",
+  "description": "Chainlink cryptocompare-test adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "cryptocompare-test"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.3.6",
```

Comment:
This was merged after work on this was done, but would you mind upgrading to `0.5.1`? That version has the noticeable refactor of the type generics, and would be good to do it before merging imo